src/Listener/ListenerOptions.php

```diff
@@ -401,7 +401,7 @@ public static function normalizePath($path)
     /** @deprecated Use self::useLaminasLoader instead */
     public function useZendLoader(): bool
     {
-        return $this->useLaminasLoader(...func_get_args());
+        return $this->useLaminasLoader();
```

Comment:
This change seems wrong: it was done for compatibility, AFAIK, and should stay.

Reply:
Ok, rolled back https://github.com/laminas/laminas-modulemanager/pull/39/commits/6f957ebd0d829c6d8db37fcbd565cc682478e57d